app/client/ui/HomeLeftPane.ts

```diff
@@ -24,6 +21,7 @@ import {Workspace} from 'app/common/UserAPI';
 import {computed, dom, domComputed, DomElementArg, observable, Observable, styled} from 'grainjs';
 import {createHelpTools, cssLeftPanel, cssScrollPane,
         cssSectionHeader, cssTools} from 'app/client/ui/LeftPanelCommon';
+import {createDocAndOpen, importDocAndOpen, importFromPluginAndOpen} from "app/client/ui/NewDocMethods";
```

Comment:
Please use locally consistent quotes. Please do your best to put imports in alphabetical order, even if prev devs have left small muddles.

Reply:
I reordered all imports here manually.

I noticed that our eslint config has `ignoreDeclarationSort: true`, which effectively disables `sort-imports` checks across lines. Is that intentional?